proto/p4/config/v1/p4info.proto

```diff
@@ -114,6 +116,72 @@ message P4Ids {
   }
 }
 
+message MvlutParam {
+  message Param {
+    uint32 param_id = 1;
+    bytes value = 2;
+  }
+  repeated Param params = 1;
+}
+
+message MvlutEntry {
+  uint32 mvlut_id = 1;
+  repeated FieldMatch match = 2;
+ MvlutParam param = 3;
+}
+
+message MatchValueLookupTable {
+  p4.config.v1.Preamble preamble = 1;
+  repeated p4.config.v1.MatchField match_fields = 2;
+  message Param {
+    uint32 id = 1;
+    string name = 2;
+    uint32 bitwidth = 3;
+  }
+  repeated Param params = 3;
+  uint32 size = 4;
+}
```

Comment:
Apart from this and the extern enum additions, I feel all the other additions to the p4info in this PR aren't required.